Colander/Classes/CalendarViewModel.swift

```diff
@@ -53,7 +53,7 @@ class CalendarViewModel {
         self.showLeadingWeeks = showLeadingWeeks
         self.showTrailingWeeks = showTrailingWeeks
         self.calendar = calendar
-        SwiftDate.defaultRegion = Region(calendar: calendar, zone: TimeZone.current, locale: Locale.current)
+        SwiftDate.defaultRegion = Region(calendar: calendar, zone: calendar.timeZone, locale: calendar.locale ?? Locale.current)
```

Comment:
Now that I've removed out custom extensions in favor of what SwiftDate provides, setting this should mean operations on `Date` (rather than `DateInRegion`) should consider the calendar as expected.